.github/workflows/coverage.yml

```diff
@@ -0,0 +1,61 @@
+# Licensed under the Apache License: http://www.apache.org/licenses/LICENSE-2.0
+# For details: https://github.com/nedbat/coveragepy/blob/master/NOTICE.txt
+
+name: "Coverage"
+
+on:
+  push:
+    branches: ["master"]
+  pull_request:
+  workflow_dispatch:
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  coverage:
+    name: "Coverage on Python ${{ matrix.python-version }}"
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        python-version:
+          - "2.7"
+          - "3.5"
+          - "3.9"
+      fail-fast: false
+
+    steps:
+      - name: "Check out the repo"
+        uses: "actions/checkout@v2"
+
+      - name: "Set up Python"
+        uses: "actions/setup-python@v2"
+        with:
+          python-version: "${{ matrix.python-version }}"
+
+      - name: "Install dependencies"
+        run: |
+          set -xe
+          python -VV
+          python -m site
+          python -m pip install -r requirements/ci.pip
+          python -m pip install -c requirements/pins.pip tox-gh-actions
+
+      - name: "Run tox for ${{ matrix.python-version }}"
+        env:
+          COVERAGE_COVERAGE: "yes"
+        run: |
+          set -xe
+          python -m tox
+          ls -al .*cov*
```

Comment:
Are these `ls` commands for debugging?

Reply:
Yes :)

Reply:
Codecov is saying my coverage went from 93.97% to 100.00%.  That isn't right.  The codecov commands are magically implicit, so I'm trying to understand what they are doing.

Reply:
If you do find out how that magic works, we need it in pbench.  =)

Reply:
My Coverage action now works better: it combines runs from different versions of Python and uploads one coverage.xml file.